### PR TITLE
feat: upgrade to PostCSS 8

### PR DESCRIPTION
**Pull request by armano2**
This PR upgrades the module to PostCSS 8 along with a few other changes:

- Upgraded ESLint to V7
- Upgraded postcss-tape to V6
- Changed the engines since PostCSS supports only Node.js 10 and up
- Travis tests run on Node.js 10 and 12
- Removed babel / rollup since all features used are now available in Node.js

see csstools/postcss-preset-env#191

closes #9, closes #10

### CODE REVIEW

File: lib/get-image.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+	!(
+		node.parent.parent &&
+		node.parent.parent.type === 'func' &&
+		imageSetFunctionMatchRegExp.test(node.parent.parent.name)
 	)
-	? String(node)
-: Object(node).type === 'string'
-	? node.value
-: false;
+		? /* node.raws.before + */ String(node)
```
Comment (armano2) [inline lib/get-image.js]: if its desired to keep previous behavior of keeping new lines this has to be uncommented

Comment (jonathantneal) [reply lib/get-image.js]: If there’s no functional trade off to preserve spacing, I’m in favor of preserving the original formatting.

Comment (armano2) [reply lib/get-image.js]: @jonathantneal i just u;dated it

File: test/basic.expect.css [hunk 2]
```diff
@@ -161,13 +161,13 @@
 
 .test-no-url {
 	order: 1;
-	background-image: img/test.png;
+	background-image: "img/test.png";
```
Comment (armano2) [inline test/basic.expect.css:164]: this is related to a bug: https://github.com/csstools/postcss-image-set-function/issues/5, we actually should wrap this node in `url` i its a `quoted` but this should be done in separate pr
